src/systems/ec/implementations/react/components/file/src/FileDownload.jsx

```diff
@@ -1,44 +1,90 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Button from '@ecl/ec-react-component-button';
 import Icon from '@ecl/ec-react-component-icon';
 import Link from '@ecl/ec-react-component-link';
 
 export const FileDownload = ({
+  variant,
   title,
   language,
   meta,
   icon,
   download,
+  detailMeta,
+  description,
+  image,
   translation,
   className,
   ...props
 }) => {
-  const classNames = classnames(className, 'ecl-file');
+  const classNames = classnames(className, 'ecl-file', {
+    [`ecl-file--${variant}`]: variant,
+  });
 
   return (
     <div {...props} className={classNames} data-ecl-file>
       <div className="ecl-file__container">
-        <Icon
-          {...icon}
-          className={classnames(icon.className, 'ecl-file__icon')}
-        />
-        <div className="ecl-file__info">
-          <div className="ecl-file__title">{title}</div>
-          <div className="ecl-file__language">{language}</div>
-          <div className="ecl-file__meta">{meta}</div>
-        </div>
-        <Link
-          {...download}
-          icon={{
-            shape: 'ui--download',
-            size: 'fluid',
-          }}
-          variant="standalone"
-          className={classnames(download.className, 'ecl-file__download')}
-          download
-        />
+        {variant === 'thumbnail' ? (
+          <Fragment>
+            <div className="ecl-file__detail">
+              <div className="ecl-file__detail-info">
+                {detailMeta && (
+                  <div className="ecl-file__detail-meta">{detailMeta}</div>
+                )}
+                <div className="ecl-file__title">{title}</div>
+                {description && (
+                  <div className="ecl-file__description">{description}</div>
+                )}
+              </div>
+
+              {image && (
```

Comment:
I think, image goes under the content too quickly (lg). Is it not possible to keep the image opposite the content longer up to sm format? 

<img width="829" alt="Screenshot 2020-07-08 at 09 18 21" src="https://user-images.githubusercontent.com/1289480/86890503-b25e3a00-c0fd-11ea-9a70-ad3541762aef.png">

seeing other variant, seeing other variants, the same behavior with the download button, they could also stay in front of the content longer, what do you think?

<img width="835" alt="Screenshot 2020-07-08 at 09 31 45" src="https://user-images.githubusercontent.com/1289480/86890619-e0dc1500-c0fd-11ea-8c64-bd23b1c2efef.png">

Reply:
Same for the preview on the showcase page, the download btn and the thumbnails goes below the content. Is it necessary to pass the contents below so early (lg)?

<img width="746" alt="Screenshot 2020-07-08 at 09 38 10" src="https://user-images.githubusercontent.com/1289480/86891222-c9515c00-c0fe-11ea-8881-314bfd288e02.png">

Reply:
I will check with the designers.
using sm breakpoint would probably break the design at some point (if the title is long for instance), but maybe a md breakpoint could be fine.